src/content/guides/development.mdx

```diff
@@ -100,7 +100,7 @@ Now let's make sure we have something to debug, so let's create an error in our
  }
 ```
 
-Run an `npm run build`, it should compile to something like this:
+Run an `yarn build`, it should compile to something like this:
```

Comment:
I think it's ok to use `npm` inside the `examples`/`content/guides`

Reply:
I'd like to ensure a consistent package manager been used?

See also https://github.com/webpack/webpack.js.org/pull/6301/files#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R10

Reply:
Nah, `examples`/`content` are user facing content. Let's not push them to package managers them might not have while `npm` is bundled with Node.js.

Reply:
> Let's not push them to package managers they might not have while npm is bundled with Node.js.

Fair enough.